lua/lspconfig/pylsp.lua

```diff
@@ -0,0 +1,27 @@
+local configs = require 'lspconfig/configs'
+
+configs.pylsp = {
+  default_config = {
+    cmd = {"pylsp"};
+    filetypes = {"python"};
+    root_dir = function(fname)
+      return vim.fn.getcwd()
```

Comment:
Add some common root markers and use `util.path.dirname(fname)` instead of `vim.fn.getcwd()`, e.g:

```suggestion
      local root_files = {
        "pyproject.toml",
        "setup.py",
        "setup.cfg",
        "requirements.txt"
      }
      return util.root_pattern(unpack(root_files))(fname) or
        util.find_git_ancestor(fname) or
        util.path.dirname(fname)
```

Reply:
Done